docs/Zeek-Customization.md

```diff
@@ -0,0 +1,72 @@
+# Zeek Customization
+
+- [Summary](#summary)
+- [Contact us!](#contact-us)
+- [Details](#details)
+
+# Summary
+
+Brim uses [Zeek](https://www.zeek.org) to generate Zeek logs from pcaps; the Zeek logs are then combined and stored in
+ [ZNG](https://github.com/brimsec/zq/blob/master/zng/docs/spec.md) format. Brim comes with a Zeek bundle
+it uses just for this process.
+
+Starting with `v0.10.0`, Brim can be configured to use a Zeek setup other than its default. This may be useful for:
+
+   * Users with an existing Zeek installation, to generate logs in Brim similar to their existing Zeek logs.
+   * Users that want to experiment with a Zeek configuration different from the Brim default.
+   * Zeek script authors that would like to use Brim as a script development aid.
+
+To use a different Zeek setup, there is now a Brim preference to specify the Zeek "runner", an executable script or 
+command run when a pcap import occurs, which launches Zeek to read the pcap's data & write Zeek logs.
+
+# Contact us!
```

Comment:
Feedback: When I was reading this, it was strange to get a Contact us before the Details. Move to the last section?

Reply:
In @alfred-landrum's defense, he seems to be following a pattern I started with https://github.com/brimsec/brim/wiki/Zeek-JSON-Import. Maybe it's a little heavyweight to put it high up in the doc, but I'm a fan of beating the "contact us" drum early & often to really reinforce that we want to talk to people, particularly if they're using these "power user" features.

Reply:
It's also a less jarring when read on the wiki page than the raw markdown text, btw, since the markdown page shows a table of contents & the 'contact us' section is small.